src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileReaderImplTest {
+    private static FileReader fileReader;
+
+    @BeforeAll
+    public static void setUp() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test
+    public void completedList_Ok() {
+        String fileName = "fruits.csv";

```

Comment:
move the path to the constant fields. Use the test/resources  directory for testing data